pages/a/snowsprint-2016.html

```diff
@@ -0,0 +1,92 @@
+title: Crate.IO Snowsprint 2016 Recap
+link: https://crate.io/blog/snowsprint-2016/
+author: Chris Ward, Johannes Moser
+description: Short social media based recap of the snowsprint
+created: 2016/01/27 18:00:00
+post_name: snowsprint-2016
+status: publish
+post_type: post
+tags: Snowsprint, Hittisau, Code, Bregenzer Wald, Skiing
+category: news
+
+
+# [Crate.IO SnowSprint 2016](/snowsprint)
+
+Every once in a year it is getting quiet in the Crate Offices in Dornbirn and Berlin, when all the people are gathering somewhere in the mountains. It's Snowsprint time.
+
+This year a couple of guys from all over the world (Azerbaidjan, Sweden, Scotland,...) are joing on us in Hittisau in the middle of the Bregenzer Wald.
+
+<blockquote class="instagram-media" data-instgrm-captioned data-instgrm-version="6" style=" background:#FFF; border:0; border-radius:3px; box-shadow:0 0 1px 0 rgba(0,0,0,0.5),0 1px 10px 0 rgba(0,0,0,0.15); margin: 1px; max-width:658px; padding:0; width:99.375%; width:-webkit-calc(100% - 2px); width:calc(100% - 2px);"><div style="padding:8px;"> <div style=" background:#F8F8F8; line-height:0; margin-top:40px; padding:50.0% 0; text-align:center; width:100%;"> <div style=" background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACwAAAAsCAMAAAApWqozAAAAGFBMVEUiIiI9PT0eHh4gIB4hIBkcHBwcHBwcHBydr+JQAAAACHRSTlMABA4YHyQsM5jtaMwAAADfSURBVDjL7ZVBEgMhCAQBAf//42xcNbpAqakcM0ftUmFAAIBE81IqBJdS3lS6zs3bIpB9WED3YYXFPmHRfT8sgyrCP1x8uEUxLMzNWElFOYCV6mHWWwMzdPEKHlhLw7NWJqkHc4uIZphavDzA2JPzUDsBZziNae2S6owH8xPmX8G7zzgKEOPUoYHvGz1TBCxMkd3kwNVbU0gKHkx+iZILf77IofhrY1nYFnB/lQPb79drWOyJVa/DAvg9B/rLB4cC+Nqgdz/TvBbBnr6GBReqn/nRmDgaQEej7WhonozjF+Y2I/fZou/qAAAAAElFTkSuQmCC); display:block; height:44px; margin:0 auto -44px; position:relative; top:-22px; width:44px;"></div></div> <p style=" margin:8px 0 0 0; padding:0 4px;"> <a href="https://www.instagram.com/p/BA9RFfvxP2M/" style=" color:#000; font-family:Arial,sans-serif; font-size:14px; font-style:normal; font-weight:normal; line-height:17px; text-decoration:none; word-wrap:break-word;" target="_blank">Hittisau. #snowsprint</a></p> <p style=" color:#c9c8cd; font-family:Arial,sans-serif; font-size:14px; line-height:17px; margin-bottom:0; margin-top:8px; overflow:hidden; padding:8px 0 7px; text-align:center; text-overflow:ellipsis; white-space:nowrap;">Ein von johannes moser (@joemoe) gepostetes Foto am <time style=" font-family:Arial,sans-serif; font-size:14px; line-height:17px;" datetime="2016-01-25T09:39:13+00:00">25. Jan 2016 um 1:39 Uhr</time></p></div></blockquote>
```

Comment:
you should improve your linebreaks and indentation in future blog posts :)
![giphy](https://cloud.githubusercontent.com/assets/158125/12649244/0944723c-c5de-11e5-9ace-516db81d79c1.gif)